### PR TITLE
Support wildcard match on search/query output fields

### LINKED ISSUE #6411: Support wildcard match on search/query output field
**Please state your issue using the following template and, most importantly, in English.**
Search and query has to specify output field, it not currently it only returns ids.

**Is your feature request related to a problem? Please describe.**
I wish Milvus could support * in output field and retrieve all the field including the vector field.

**Describe the solution you'd like**
add * support in output field. * should not be a valid field name and should be banned at sdk.
we need better document to explain  output field.

**Describe alternatives you've considered**
we can use nil, or the default value as select * , but it maybe confusing and causing performance degradation.
thus we import * directly, it should be easier to understand, and default behavior is not changed.

**Additional context**
Add any other context, code examples, or references to existing implementations about the feature request here.

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>

issue: #6411

### CODE REVIEW

File: internal/proxy/task.go [hunk 1]
```diff
@@ -1337,6 +1337,23 @@ func (st *SearchTask) getVChannels() ([]vChan, error) {
 	return st.chMgr.getVChannels(collID)
 }
 
+// https://github.com/milvus-io/milvus/issues/6411
+// Support wildcard match
+func translateOutputFields(outputFields []string, schema *schemapb.CollectionSchema) ([]string, error) {
```
Comment (czs007) [inline internal/proxy/task.go:1342]: Please do a trim operation on all strings, so that "  *   " and  "*" should have the same meaning

Comment (DragonDriver) [reply internal/proxy/task.go:1342]: OK, I will fix this soon.

Comment (DragonDriver) [reply internal/proxy/task.go:1343]: Fixed and related test cases was also added.

File: internal/proxy/task.go [hunk 2]
```diff
@@ -1337,6 +1338,23 @@ func (st *SearchTask) getVChannels() ([]vChan, error) {
...
+	if len(outputFields) == 1 && strings.Trim(outputFields[0], " ") == "*" {
```
Comment (czs007) [inline internal/proxy/task.go:1344]: better to use strings.TrimSpace, as "\t \n * \r\t" should also be "*" ? what do you think?

Comment (DragonDriver) [reply internal/proxy/task.go:1344]: Yeah, my fault here. I test strings.TrimSpace and strings.Trim(s, " "), two are equal, but strings.TrimSpace is more clear.

**Pull request merged**